### ISSUE TITLE
[SDK52] Can't install dependencies for new project using SDK52

### ISSUE THREAD

**Issue by SzymonGalazka** [expo/expo#32447]
### Summary

I wanted to try the new beta sdk. The command provided in [this article](https://expo.dev/changelog/2024/10-24-sdk-52-beta) `npx create-expo-app --template default@beta` is unable to install its initial dependencies.

```bash
 npx create-expo-app --template default@beta
✔ What is your app named? … test
✔ Downloaded and extracted project files.
> npm install
npm warn ERESOLVE overriding peer dependency
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: @react-navigation/bottom-tabs@7.0.0-rc.33
npm warn Found: peer react-native-screens@">= 4.0.0" from @react-navigation/bottom-tabs@7.0.0-rc.33
npm warn node_modules/@react-navigation/bottom-tabs
npm warn   @react-navigation/bottom-tabs@"7.0.0-rc.33" from expo-router@4.0.0-preview.4
npm warn   node_modules/expo-router
npm warn
npm warn Could not resolve dependency:
npm warn peer react-native-screens@">= 4.0.0" from @react-navigation/bottom-tabs@7.0.0-rc.33
npm warn node_modules/@react-navigation/bottom-tabs
npm warn   @react-navigation/bottom-tabs@"7.0.0-rc.33" from expo-router@4.0.0-preview.4
npm warn   node_modules/expo-router
npm warn ERESOLVE overriding peer dependency


npm warn ERESOLVE overriding peer dependency
npm error code ETARGET
npm error notarget No matching version found for react-native-screens@>= 4.0.0.
npm error notarget In most cases you or one of your dependencies are requesting
npm error notarget a package version that doesn't exist.
npm error A complete log of this run can be found in: /Users/sg/.npm/_logs/2024-10-29T14_47_59_037Z-debug-0.log
Something went wrong installing JavaScript dependencies. Check your npm logs. Continuing to create the app.
Error: npm install exited with non-zero code: 1
```

![Zrzut ekranu 2024-10-29 o 15 53 26](https://github.com/user-attachments/assets/bdfc3140-aada-4a6c-8529-1a2de9d0dfa1)

### SDK Version

52

### Environment

```text
expo-env-info 1.2.0 environment info:
    System:
      OS: macOS 15.0.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 22.10.0 - /opt/homebrew/bin/node
      npm: 10.9.0 - /opt/homebrew/bin/npm
      Watchman: 2024.10.21.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.15.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 24.0, iOS 18.0, macOS 15.0, tvOS 18.0, visionOS 2.0, watchOS 11.0
    IDEs:
      Xcode: 16.0/16A242d - /usr/bin/xcodebuild
    Expo Workflow: managed
```

### Minimal reproducible example

.

**Comment by suhailk-k** [comment]
use yarn 
`yarn create expo-app --template default@beta`

**Comment by simonbengtsson** [comment]
I solved it by adding this override to package.json

```json
  "overrides": {
    "react-native-screens": "4.0.0-beta.14"
  },
```

**Comment by entiendoNull** [comment]
Duplicate of this issue: https://github.com/expo/expo/issues/32389

Reply from Expo team member:
> [...] This error is happening because of @react-navigation/bottom-tabs v7, which internally depends on the beta version of react-native-screens, but we have already contacted the react-navigation team and this will be resolved soon.
> 
> Here is some additional context on how npm handle version comparisons and why the RC versions of react-native-screens are not being resolved correctly [npm/npm#8854](https://github.com/npm/npm/issues/8854)
> 
> For now you can fix this by running `npm install --legacy-peer-deps`, use the `--force` flag or using a different package manager like yarn or pnpm

**Comment by brentvatne** [comment]
fixed with #32447
